README.md

```diff
@@ -1,6 +1,6 @@
 # Capistrano Remote
 
-Every so often you need to look at your production data or otherwise run some manual maintenance tasks in your production Rails application.
+Ever so often you need to look at your production data or otherwise run some manual maintenance tasks in your production Rails application.
```

Comment:
So I've been doing it wrong / American all this time? Awesome :fireworks: 

Nevery mind me then :)